src/main/java/roomescape/repository/ReservationRepository.java

```diff
@@ -75,7 +76,11 @@ default void updateFirstWaiting(Reservation requestedReservation) {
         findByDateAndThemeAndTime(requestedReservation.getDate(),
                 requestedReservation.getTheme(),
                 requestedReservation.getReservationTime()
-        ).ifPresent(Reservation::book);
+        ).ifPresent(reservation -> {
```

Comment:
메서드 체이닝을 사용할거라면 일관되게 사용하는게 가독성이 좋은것 같아요

```suggestion
        findByDateAndThemeAndTime(requestedReservation.getDate(),
                requestedReservation.getTheme(),
                requestedReservation.getReservationTime())
                .filter(reservation -> reservation.getReservationStatus() == ReservationStatus.WAITING)
                .ifPresent(Reservation::book);
```

Reply:
`Optional.filter()` 메서드를 까맣게 잊고 있었네요 😮 
메서드 체이닝 통일로 가독성이 훨씬 좋아질 수 있었네요!